freqtrade/resolvers/strategy_resolver.py

```diff
@@ -122,7 +120,7 @@ def _load_strategy(
         current_path = Path(__file__).parent.parent.joinpath('strategy').resolve()
 
         abs_paths = [
-            config['user_data_dir'].joinpath('strategies'),
+            Path(config['user_data_dir']).joinpath('strategies'),
             current_path,
         ]
```

Comment:
I'm not sure why this is necessary:
```python
strategy = StrategyResolver({'user_data_dir': Path(config['user_data_dir']),
                       'strategy_path': Path(config['strategy_path'])}).strategy
```
Given that `config = Configuration.from_files(config_files)`, shouldn't 
`config['user_data_dir']` == Path object?

But when I run:
```python
strategy = StrategyResolver(config=config).strategy
```

I get:
```python
    122         abs_paths = [
--> 123             config['user_data_dir'].joinpath('strategies'),
    124             current_path,
    125         ]

AttributeError: 'str' object has no attribute 'joinpath'
```